### PR TITLE
LPS-168298 Use correct key in getCreationMenu method

### PR DESCRIPTION
**Pull request by vngoliveira**
Related issue: [LPS-168298](https://issues.liferay.com/browse/LPS-168298)

### CODE REVIEW

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/ViewOrganizationsManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public CreationMenu getCreationMenu() throws PortalException {
 								organizationType);
 							dropdownItem.setLabel(
 								LanguageUtil.get(
-									_httpServletRequest, organizationType));
+									_httpServletRequest, "add-organization"));
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/ViewOrganizationsManagementToolbarDisplayContext.java:114]: I have concerns about this change. By default, there is only one type of organization, but if there is more than one type, then this change will render multiple menu items with the same label, which will be confusing. Perhaps we can introduce a parameterized language key (if it does not exist already) that accepts the type, something like `add-x`. Thoughts?

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/ViewOrganizationsManagementToolbarDisplayContext.java (hunk 1 above)
Comment (vngoliveira) [reply modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/ViewOrganizationsManagementToolbarDisplayContext.java]: I think I understood. I did a change following the structure in another similar file.

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:stable

**Comment by vngoliveira** [comment]
ci:test:relevant

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@vngoliveira thank you for updating this. Will you please post a before-and-after pair of screenshots? I want to make sure this is really what we want. Thanks!

**Comment by vngoliveira** [comment]
Before
screenshot-1677603993_360
After
screenshot-1677603459_360

**Review by drewbrokke — APPROVED** [review]
Looks great, thank you @vngoliveira!

**Comment by drewbrokke** [comment]
ci:forwar

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
@patriciajeanperez are there any functional test updates needed for this one?

**Comment by patriciajeanperez** [comment]
ci:test:users

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Comment by patriciajeanperez** [comment]
ci:stop

**Pull request closed without merging**